File: C#/bumpsInTheRoad.cs
using System;
using System.Linq;

class Kata
{
    public static string Bump(string input)
    {
        var bumps = input.Count(c => "n".Contains(c));
        
        if (bumps <= 15)
          return "Woohoo!";
        return "Car Dead";
    }
}

File: Java/bumpsInTheRoad.java
public class BumpsTheRoad {
  public static String bumps(final String road) {
    int bumpCounter = 0;
    char[] roadArray = road.toCharArray();
    
    for (char letter: roadArray) {
      if (letter == 'n')
        bumpCounter++;
    }
    
    if (bumpCounter > 15)
      return "Car Dead";
    return "Woohoo!";
  }
}